Error for the multi threads compatibility

The `Result` is unable to be sent because `Send` is absent.
So the `request` function cannot be called on multi threads runtime.
```
error[E0277]: `dyn StdError` cannot be sent between threads safely
```

```rs
let response = top_api.request(ApiName::GenerateAffiliateLinks, Some(request_parameters)).await?;
```

--- a/src/iop.rs
+++ b/src/iop.rs
@@ -101,7 +101,7 @@ impl std::fmt::Display for IopError {
 
 impl std::error::Error for IopError {}
 
-pub type Result<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+pub type Result<T> = std::result::Result<T, Box<dyn std::error::Error + Send>>;
 
 #[allow(dead_code)]
 #[derive(PartialEq)]

